Examples/Example-iOS-ObjC/Info.plist

```diff
@@ -15,9 +15,9 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>1.0</string>
```

Comment:
is this something that we should replace with 1.0 version? I think they're overridden by cicd?

Reply:
Restored. Changed coz Simulator was shouting about bundle version.

Reply:
Don’t do this :) read the original commit statement when I added this. Should not be a hardcoded version. Make sure you’re on a recent xcode version. Leave it on MARKETING_VERSION.